reset_killed: mutex: LOCK_thd_data (0x62b00015ffe8) locking
reset_killed: mutex: LOCK_thd_data (0x62b00015ffe8) locked
reset_killed: mutex: LOCK_thd_data (0x62b00015ffe8) unlocking
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e62a8
dispatch_command: mutex: LOCK_thd_data (0x62b00015ffe8) locking
dispatch_command: mutex: LOCK_thd_data (0x62b00015ffe8) locked
dispatch_command: mutex: LOCK_thd_data (0x62b00015ffe8) unlocking
dispatch_command: query: SET DEBUG_DBUG = '+d,innodb_OOM_prepare_inplace_alter';
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e6348
LEX::start: info: This: 0x62b0001625b8 thd_arg->lex: 0x62b0001625b8
st_select_lex_node::set_linkage: info: node: 0x62b000162e88  linkage: 0->0
st_select_lex_node::set_linkage: info: node: 0x62b000162690  linkage: 4->0
st_select_lex_node::set_linkage: info: node: 0x62b000162690  linkage: 0->4
reset_current_stmt_binlog_format_row: debug: temporary_tables: no, in_sub_stmt: no, system_thread: NON_SYSTEM_THREAD
LEX::main_select_push: info: service: 0
LEX::push_select: info: Top Select was 0x0 (0)  depth: 0  pushed: 0x62b000162e88 (1)
LEX::push_context: info: Context: 0x62b000162ed0 Select: 0x62b000162e88 (1)
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e6390
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e63a8
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e63d8
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e6450
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e64c8
find_sys_var: enter: var 'DEBUG_DBUG'
my_hash_first_from_hash_value: exit: found key at 159
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e64f0
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e6538
LEX::pop_select: info: Top Select is 0x0 (0)  depth: 0  poped: 0x62b000162e88 (1)
LEX::pop_context: info: Context: 0x62b000162ed0 Select: 0x62b000162e88 (1)
LEX::pop_select: info: Top Select is empty -> sel builtin: 0x62b000162e88  service: 0
st_select_lex::set_nest_level: enter: select #1 0x62b000162e88 nest level: 0
st_select_lex::check_parameters: enter: select #1 0x62b000162e88 nest level: 0
mysql_parse: mutex: LOCK_thd_data (0x62b00015ffe8) locking
mysql_parse: mutex: LOCK_thd_data (0x62b00015ffe8) locked
mysql_parse: mutex: LOCK_thd_data (0x62b00015ffe8) unlocking
open_and_lock_tables: enter: derived handling: 1
open_tables: THD::enter_stage: Opening tables at /root/mariadb/sql/sql_base.cc:4222
open_tables: THD::enter_stage: After opening tables at /root/mariadb/sql/sql_base.cc:4505
open_tables: THD::enter_stage: (null) at /root/mariadb/sql/sql_base.cc:4506
open_tables: open_tables: returning: 0
mysql_handle_derived: enter: phases: 0x1
alloc_root: enter: root: 0x62b0001641a8
alloc_root: exit: ptr: 0x6290000e6550
2022-08-06  3:15:12 3 [ERROR] Invalid roles_mapping table entry user:'u@localhost', rolename:'%'
2022-08-06  3:15:14 3 [Warning] Aborted connection 3 to db: 'awrifhwh' user: 'root' host: 'localhost' (Query was empty)
2022-08-06  3:15:38 4 [ERROR] Invalid roles_mapping table entry user:'u@localhost', rolename:'%'
2022-08-06  3:15:38 4 [ERROR] Invalid roles_mapping table entry user:'r_sel@', rolename:'r_rld'
2022-08-06  3:15:38 4 [ERROR] Invalid roles_mapping table entry user:'r_sel@', rolename:'r_del'
2022-08-06  3:15:38 4 [ERROR] Invalid roles_mapping table entry user:'r_del@', rolename:'r_ins'
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `ovvgssyk`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `g` in table `ovvgssyk`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `fgaxcmsh`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `g` in table `fgaxcmsh`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `ozqeczho`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `g` in table `ozqeczho`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `lopkdbst`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `g` in table `lopkdbst`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `wnzrhxbo`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:38 4 [ERROR] InnoDB: Flagged corruption of `g` in table `wnzrhxbo`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `fclioepe`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `g` in table `fclioepe`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `fgjkhqys`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `g` in table `fgjkhqys`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `gnelrwep`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `g` in table `gnelrwep`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `fid` in table `ldgsciji`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [ERROR] InnoDB: Flagged corruption of `g` in table `ldgsciji`.`gis_point` in dict_set_index_corrupted
2022-08-06  3:15:39 4 [Note] Deleted Master_info file '/root/out/mysql_client_dir_2000/master.info'.
2022-08-06  3:15:39 4 [Note] Deleted Master_info file '/root/out/mysql_client_dir_2000/relay-log.info'.
2022-08-06  3:15:45 4 [Note] Deleted Master_info file '/root/out/mysql_client_dir_2000/master.info'.
2022-08-06  3:15:45 4 [Note] Deleted Master_info file '/root/out/mysql_client_dir_2000/relay-log.info'.
2022-08-06  3:15:47 4 [Note] Master connection name: ''  Master_info_file: 'master.info'  Relay_info_file: 'relay-log.info'
2022-08-06  3:15:47 4 [Note] 'CHANGE MASTER TO executed'. Previous state master_host='', master_port='3306', master_log_file='', master_log_pos='4'. New state master_host='', master_port='3306', master_log_file='', master_log_pos='4'.
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `rfoiuimq`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './rfoiuimq/gis_polygon'
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `xikaudin`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './xikaudin/gis_polygon'
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `icecnbqf`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './icecnbqf/gis_polygon'
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `qoihjshb`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './qoihjshb/gis_polygon'
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `xyluiamf`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './xyluiamf/gis_polygon'
2022-08-06  3:15:50 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `iwhfzpsk`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:50 4 [ERROR] Got error 180 when reading table './iwhfzpsk/gis_polygon'
2022-08-06  3:15:51 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `tyoiaenq`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:51 4 [ERROR] Got error 180 when reading table './tyoiaenq/gis_polygon'
2022-08-06  3:15:51 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `sxxeqghp`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:51 4 [ERROR] Got error 180 when reading table './sxxeqghp/gis_polygon'
2022-08-06  3:15:51 4 [ERROR] InnoDB: Flagged corruption of `gis_multi_linestring` in table `gsjqsvxd`.`gis_polygon` in dict_set_index_corrupted
2022-08-06  3:15:51 4 [ERROR] Got error 180 when reading table './gsjqsvxd/gis_polygon'
SIGKILL myself
